docs/api.rst

```diff
@@ -5956,6 +6017,11 @@ ActionRow
 
 .. autoclass:: disnake.ui.ActionRow
     :members:
+    :exclude-members: add_select
+
+    .. method:: add_select
+
+        Alias of :func:`~disnake.ui.ActionRow.add_string_select`.
```

Comment:
```suggestion
        .. deprecated:: 2.7

        Alias of :func:`~disnake.ui.ActionRow.add_string_select`.
```

Reply:
Should the old methods/types really be deprecated? As of right now everything else is just aliased with no warning, and I'm personally fine with that (could change it in 3.0, perhaps).
I feel like the general perception is that the new selects are specialized forms of the "classic" select, instead of completely independent types, so there's still this notion of "a select menu" being the classic string select menu. Maybe that's just me tho.

Reply:
I think this should be deprecated but no plans to remove them until a major version bump (or extended time on 2.x)

My future plan is this method can act as a factory for any of the currently four select types.